websites/D/Discovery+/metadata.json

```diff
@@ -0,0 +1,56 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "638080361179512853",
+		"name": "Dark_Ville#2756"
+	},
+	"service": "Discovery+",
+	"description": {
+		"en": "Watch HGTV, Food Network, TLC, ID and more plus exclusive originals, all in one place. Start your 7-day free trial today.",
+		"nl": "Welkom bij discovery+, the home of real-life entertainment en sport. Abonneer nu en volg volledige seizoenen, bekijk programma’s alvast vooruit of kijk terug wat je hebt gemist."
```

Comment:
Official description for the dutch version. So yes it doesn't match but both site version have different descriptions.